tests/src/Kernel/Access/WikiTest.php

```diff
@@ -129,12 +130,14 @@ public function setUp() {
 
     // Grant both members and non-members permission to edit any group content.
     foreach ([OgRoleInterface::AUTHENTICATED, OgRoleInterface::ANONYMOUS] as $role_name) {
```

Comment:
Checking we throw an exception can be added into the existing Kernel tests (we could have done it in Unit, but it's already written as Kernel). See [this](https://github.com/Gizra/og/blob/438869c2928e6eed4dc0dc005c92dcca87cb5b92/tests/src/Kernel/Entity/OgMembershipTest.php#L126) for example, as a handy way to tell that an exception is expected.